app/src/main/java/io/left/meshim/models/Message.java

```diff
@@ -137,10 +146,12 @@ public Message(User sender, User recipient, String message) {
      * @param isMyMessage if this device's user sent the message
      */
     @Ignore
-    public Message(User sender, User recipient, String message, boolean isMyMessage) {
+    public Message(User sender, User recipient, String message, boolean isMyMessage, String filePath, String fileName) {
```

Comment:
If having a `filePath` and `fileName` is optional, I would keep the old constructor around and have it call this one with default values.